pkg/specgen/generate/storage.go

```diff
@@ -427,3 +432,31 @@ func InitFSMounts(mounts []spec.Mount) error {
 	}
 	return nil
 }
+
+func addReadWriteTmpfsMounts(mounts map[string]spec.Mount, volumes []*specgen.NamedVolume) map[string]spec.Mount {
+	readonlyTmpfs := []string{"/tmp", "/var/tmp", "/run"}
+	options := []string{"rw", "rprivate", "nosuid", "nodev", "tmpcopyup"}
+	for _, dest := range readonlyTmpfs {
+		for _, m := range mounts {
```

Comment:
From the last line of the outer loop, I can see that the `dest` field is also used as the key in the `mounts` map. If so, can you replace this loop with checking if `dest` exists in the map?

Reply:
Nice catch.